terraform/modules/grafana-stack/.gitignore

```diff
@@ -0,0 +1 @@
+configs/prometheus.yml
```

Comment:
I think the file was created with the path `/configs/grafana-datasources/prometheus.yml` - https://github.com/drim-dev/logging-benchmarks/blob/ddefc56ae19176456a96d620d03593c06833fc8a/terraform/modules/grafana-stack/main.tf#L48
But I see you deleted `/configs/prometheus.yml`. I'm not sure what it is.